api_types.ts

```diff
@@ -828,6 +828,23 @@ export enum PermissionSyncMode {
   PermissionAware = 'PermissionAware',
 }
 
+/**
+ * The type of content being indexed, which determines how the property is processed and queried.
+ * Messaging: Chat or instant messaging content
```

Comment:
Move these description to tsdoc comments on each enum value individually. It makes it harder to miss one in the future, and still shows up nice in the generated docs.